### PR TITLE
Do not select all rules for the srgmaps tables

### PR DESCRIPTION
**Pull request by isimluk**
Pick only those that are in selected in relevant profile. The profile defaults to `stig`.

### CODE REVIEW

**Comment by redhatrises** [comment]
@openscap-ci add to whitelist

File: cmake/SSGCommon.cmake [hunk 1]
```diff
@@ -1059,7 +1059,7 @@ macro(ssg_build_html_srgmap_tables PRODUCT DISA_SRG_TYPE)
         OUTPUT "${CMAKE_BINARY_DIR}/tables/table-${PRODUCT}-srgmap-flat.html"
         COMMAND "${CMAKE_COMMAND}" -E make_directory "${CMAKE_BINARY_DIR}/tables"
         # We need to use xccdf-linked.xml because ssg-${PRODUCT}-xccdf.xml has the srg_support Group removed
-        COMMAND "${XSLTPROC_EXECUTABLE}" --stringparam flat "y" --stringparam map-to-items "${CMAKE_CURRENT_BINARY_DIR_NO_SPACES}/xccdf-linked.xml" --output "${CMAKE_BINARY_DIR}/tables/table-${PRODUCT}-srgmap-flat.html" "${CMAKE_CURRENT_SOURCE_DIR}/transforms/table-srgmap.xslt" "${DISA_SRG_REF}"
+        COMMAND "${XSLTPROC_EXECUTABLE}" --stringparam profileId "${PROFILE_ID}"--stringparam flat "y" --stringparam map-to-items "${CMAKE_CURRENT_BINARY_DIR_NO_SPACES}/xccdf-linked.xml" --output "${CMAKE_BINARY_DIR}/tables/table-${PRODUCT}-srgmap-flat.html" "${CMAKE_CURRENT_SOURCE_DIR}/transforms/table-srgmap.xslt" "${DISA_SRG_REF}"
```
Comment (shawndwells) [inline cmake/SSGCommon.cmake]:
```suggestion
        COMMAND "${XSLTPROC_EXECUTABLE}" --stringparam profileId "${PROFILE_ID}" --stringparam flat "y" --stringparam map-to-items "${CMAKE_CURRENT_BINARY_DIR_NO_SPACES}/xccdf-linked.xml" --output "${CMAKE_BINARY_DIR}/tables/table-${PRODUCT}-srgmap-flat.html" "${CMAKE_CURRENT_SOURCE_DIR}/transforms/table-srgmap.xslt" "${DISA_SRG_REF}"
```

Comment (isimluk) [reply cmake/SSGCommon.cmake]: nice catch!

**Comment by shawndwells** [comment]
+cc @tedbrunell if you pull the latest code, the tables now reflect these changes

**Comment by shawndwells** [comment]
Thanks @isimluk !

**Pull request merged**